.github/workflows/OCV-Coverage-4.x-U20.yaml

```diff
@@ -0,0 +1,267 @@
+name: OCV Coverage:4.x U20
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/OCV-Coverage-4.x-U20.yaml'
+  workflow_call:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+env:
+  EXTRA_CMAKE_OPTIONS: '-DBUILD_DOCS=ON -DPYTHON_DEFAULT_EXECUTABLE=/usr/bin/python3 -DOPENCV_DOWNLOAD_PATH=/home/ci/binaries_cache -DBUILD_EXAMPLES=ON -DOPENCV_ENABLE_NONFREE=ON -DENABLE_COVERAGE=ON'
+  PR_AUTHOR: ${{ github.event.pull_request.user.login }}
+  PR_AUTHOR_FORK: ${{ github.event.pull_request.head.repo.full_name }}
+  SOURCE_BRANCH_NAME: ${{ github.head_ref }}
+  TARGET_BRANCH_NAME: ${{ github.base_ref }}
+  ANT_HOME: '/usr/share/ant'
+  GIT_CACHE_DOCKER: '/home/ci/git_cache'
+  PYTHONPATH: /build/python_loader:$PYTHONPATH
+  OPENCV_TEST_DATA_PATH: '/opencv_extra/testdata'
+  OPENCV_DOCKER_WORKDIR: '/opencv'
+  DNN_MODELS: '/home/ci/dnn-models'
+
+jobs:
+  BuildAndTest:
+    runs-on: opencv-cn-lin-x86-64
+    timeout-minutes: 1200
+    defaults:
+      run:
+        shell: bash
```

Comment:
> `Parallel framework: pthreads (nthreads=40)`

It makes sense to limit number of parallel threads (this is critical as coverage may use extra synchronization points), e.g. 2.

Please refer on this run (i5-6600 is used): https://pullrequest.opencv.org/buildbot/builders/master_coverage-lin64-debug/builds/100000

Reply:
Thank you for the suggestion! Changed to 2.

Also adapted a workflow to buildbot (changed coverage steps, removed unused tests).

Do we need to use extended DNN models testing or not? It wastes a lot of time (~1h 30m).

Reply:
> `ninja -j2`

The note above is not about the build stage. It is about the tests launching.
Use `--test_threads=2` / `--perf_threads=2` or environment `OPENCV_FOR_THREADS_NUM=2`

Check result in tests output header.

Reply:
Removed `-j2` for the build and added `--test_threads=2` for tests.